src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/AccentColorHelper.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Windows.Media;
+using System.Windows.Appearance;
+
+using MS.Internal;
+using Microsoft.Win32;
+using System.Runtime.InteropServices;
+using MS.Internal.WindowsRuntime.Windows.UI.ViewManagement;
+
+
+namespace System.Windows;
+
+internal static class AccentColorHelper
+{
+
+    #region Internal Methods
+
+    internal static Color GetAccentColor(UISettingsRCW.UIColorType uiColorType = UISettingsRCW.UIColorType.Accent)
+    {
+        Color color = _defaultAccentColor;
+
+        if (_UISettings.TryGetColorValue(uiColorType, out color))
+        {
+            return color;
+        }
+
+        return color;
```

Comment:
I believe color would be overwritten here in both cases, and the initialization is not necessary in the success case. I suggest removing line 20, doing `out Color color` and 27 just `return _defaultAccentColor`

Reply:
Yeah, agreed.